packages/spec/src/components/accordion/rules.js

```diff
@@ -0,0 +1,343 @@
+          return new ValidationError([
+            'Expected aria-expanded to be false after the heading is collapsed',
+          ]);
+        }
+      }
+    },
+  },
+
+  {
+    id: 'accordion.header.interaction.move.keyboard.tab',
+    description:
+      'Moves focus to the next focusable element; all focusable ' +
+      'elements in the accordion are included in the page Tab sequence.',
+    reference: 'https://www.w3.org/TR/wai-aria-practices/#accordion',
+    level: 'error',
+    context: defaultContext,
+    validate(root, context) {
+      const headers = root.querySelectorAll('.bx--accordion__heading');
+
+      for (let i = 0; i < context.children.length; i++) {
+        pressTab(document.body);
+        if (document.activeElement !== headers[i]) {
+          return new ValidationError([
+            'Expected elements in accordion to follow page Tab sequence.',
+          ]);
+        }
+      }
+    },
+  },
+
+  {
+    id: 'accordion.interaction.movement.keyboard.shift+tab',
+    description:
+      'Moves focus to the previous focusable element; all focusable elements ' +
+      'in the accordion are included in the page Tab sequence.',
+    reference: 'https://www.w3.org/TR/wai-aria-practices/#accordion',
+    level: 'error',
+    context: defaultContext,
+    validate(root, context) {
+      const headers = root.querySelectorAll('.bx--accordion__heading');
+
+      for (let i = context.children.length - 1; i >= 0; i--) {
+        pressShiftTab();
+        if (document.activeElement !== headers[i]) {
+          return new ValidationError([
+            'Expected elements in accordion to follow page Tab sequence.',
+          ]);
+        }
+      }
+    },
+  },
+
+  {
+    id: 'accordion.interaction.movement.keyboard.down-arrow',
+    description:
+      'If focus is on an accordion header, moves focus to ' +
+      'the next accordion header. If focus is on the last accordion header, ' +
+      'either does nothing or moves focus to the first accordion header.',
+    reference: 'https://www.w3.org/TR/wai-aria-practices/#accordion',
+    level: 'error',
+    context: defaultContext,
+    validate(root) {
+      const headers = root.querySelectorAll('.bx--accordion__heading');
+
+      pressTab();
+
+      for (let i = 1; i < headers.length; i++) {
+        press(document.activeElement, 'ArrowDown');
+        if (headers[i] !== document.activeElement) {
+          return new ValidationError([
+            'Expected ArrowDown to move focus to the next accordion header',
+          ]);
+        }
+      }
+    },
+  },
+
+  // TODO:
+  // UpArrow
+  // Home/End
+
+  {
+    id: 'accordion.header.target.size',
+    description:
+      'The target size for triggering an accordion header should ' +
+      'meet the minimum size requirement of 40px by 40px',
+    reference: 'TODO',
+    level: 'error',
+    context: defaultContext,
+    validate() {
+      // TODO: would need to be run in a real browser that implements a layout
```

Comment:
WH has multiple examples of running CSS tests within a Jest environment 

* https://github.com/scottnath/carbon-spec/pull/2
* https://github.ibm.com/Watson-Health/pal-react-example

Reply:
Not sure I understand, in this case we need a layout engine to get computed `width`/`height` properties. After some quick exploration, this seemed like something JSDOM couldn't support.

Reply:
I had some tests in our accordion POC we submitted to carbon spec that checked for CSS properties but now that I look back on it, we had to comment them out.  Scott has a POC that we were able to get CSS to load in JSDOM, but you're right that we couldn't get computed values.